playbooks/de-deploy-services.yaml

```diff
@@ -180,6 +180,24 @@
     - role: util-notify-chat
       msg: ":heavy_check_mark: Done deploying saved-searches"
 
+- name: Redeploy tagger
+  hosts: tagger:&systems
+  become: true
+  gather_facts: false
+  tags:
+    - services
+    - colored
+    - tagger
+  roles:
+    - role: util-notify-chat
+      msg: "Deploying tagger"
+    - role: de-deploy-service
+      deploy_use_color: "use_color|default(false)"
+      has_configs: "{% if use_consul_configs %}{{ false }}{% else %}{{ true }}{% endif %}"
```

Comment:
Too bad you can't just print the `true/false` value of `use_consul_configs` here 😖